bot/cogs/watchchannels/bigbrother.py

```diff
@@ -90,10 +100,8 @@ def __init__(self, bot: Bot) -> None:
 
         await ctx.send(msg)
 
-    @bigbrother_group.command(name='unwatch', aliases=('uw',))
-    @with_role(*MODERATION_ROLES)
-    async def unwatch_command(self, ctx: Context, user: FetchedMember, *, reason: str) -> None:
-        """Stop relaying messages by the given `user`."""
+    async def apply_unwatch(self, ctx: Context, user: FetchedMember, reason: str, banned: bool = False) -> None:
+        """Handles the actual user removal from the watch list."""
```

Comment:
This assumes the parameter was renamed.
```suggestion
        """
        Remove `user` from watched users and mark their infraction as inactive with `reason`.

        If `send_message` is True, a message indicating the result of the operation is sent to
        `ctx`.
        """
```